en/lessons/ecto/basics.md

```diff
@@ -1,5 +1,5 @@
 ---
-version: 2.1.1
+version: 2.1.2
```

Comment:
@tajchumber should we bump the patch or the minor here?

Reply:
I think it's a good idea :)

Reply:
@nelsonic could we make this `2.2.0`? That will allow translators to see they need to apply to change (which you can feel free to do in a follow-up PR!)

Reply:
@doomspork makes sense. updating now. 👍